src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuotes.tsx

```diff
@@ -142,25 +225,79 @@ export const TradeQuotes: React.FC<TradeQuotesProps> = memo(({ isLoading, onBack
   }, [isTradeQuoteRequestAborted, loadingSwappers, onBack])
 
   return (
-    <Box position='relative'>
+    <Flex position='relative' minHeight='full'>
       <Flex
         flexDir='column'
         width='full'
         px={2}
         pt={0}
-        pb={4}
         transitionProperty='max-height'
         transitionDuration='0.65s'
         transitionTimingFunction='ease-in-out'
         gap={2}
+        minHeight='full'
       >
-        <LayoutGroup>
-          <AnimatePresence>
-            {quotes}
-            {fetchingSwappers}
-          </AnimatePresence>
-        </LayoutGroup>
+        <Flex flexDirection='column' gap={2} flexGrow='1'>
+          <LayoutGroup>
+            <AnimatePresence>
+              {quotes}
+              {fetchingSwappers}
+            </AnimatePresence>
+          </LayoutGroup>
+
+          {!quotes.length && !fetchingSwappers.length ? (
+            <Flex height='100%' whiteSpace='normal' alignItems='center' justifyContent='center'>
+              <Flex
+                maxWidth='300px'
+                textAlign='center'
+                flexDirection='column'
+                justifyContent='center'
+                alignItems='center'
+                fontFamily={shouldUseComisSansMs ? 'Comic Sans Ms, sans-serif' : 'inherit'}
+              >
+                {!shouldUseComisSansMs ? (
+                  <PathIcon color='subtle' boxSize={10} fill='none' />
+                ) : (
+                  <Icon as={FaDog} color='text.subtle' boxSize={10} />
+                )}
+                <Text translation='trade.noQuotesAvailable' mt={2} />
+                <Text translation='trade.noQuotesAvailableDescription' color='text.subtle' />
+              </Flex>
+            </Flex>
+          ) : null}
+        </Flex>
+
+        <Accordion allowMultiple ref={unavailableAccordionRef}>
+          <AccordionItem borderBottom='none' borderTop='1px solid' borderColor='border.base'>
+            <h2>
+              <AccordionButton color='text.subtle'>
+                <Box as='span' flex='1' textAlign='left'>
+                  {translate('trade.unavailableQuotes')}
```

Comment:
preferably-blocking: this is confusing to read, do we mean invalid or unavailable?
i.e the copy is "Available/Unavailable" whereas the terminology used around this PR is "valid/invalid" which is v. different.

Let's unify this to a single vernacular.